packages/commands/src/index.ts

```diff
@@ -1266,7 +1289,12 @@ export namespace CommandRegistry {
     if (event.metaKey && Platform.IS_MAC) {
       mods.push('Cmd');
     }
-    mods.push(key);
+    if (!layout.isModifierKey(key)) {
+      mods.push(key);
+      // for  modifier and character key strings
+      return mods.join(' ');
```

Comment:
```suggestion
```

Unnecessary, this is the same statement as below

Reply:
I have removed it